src/app/modules/community-core/interceptor/interceptor.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
```

Comment:
We planned to rename the module as portal-core and portal shared

Reply:
renamed